qp_deblur/tests/test_deblur.py

```diff
@@ -199,16 +245,21 @@ def test_deblur_demux(self):
              (join(out_dir, 'deblur_out', 'deblured', 'all.seqs.fa'),
               'preprocessed_fasta')], ainfo[0].files)
 
+        file_tree = join(out_dir, 'deblur_out',
+                         'deblured', 'insertion_tree.relabelled.tre')
         self.assertEqual(
             [(join(out_dir, 'deblur_out', 'deblured', 'reference-hit.biom'),
               'biom'),
              (join(out_dir, 'deblur_out', 'deblured',
                    'reference-hit.seqs.fa'), 'preprocessed_fasta'),
-             (join(out_dir, 'deblur_out', 'deblured',
-                   'insertion_tree.relabelled.tre'),
-              'plain_text')
+             (file_tree, 'plain_text')
              ], ainfo[1].files)
 
+        # finally we are gonna test that the patch is added to the tree
+        with open(file_tree, 'r') as tree_fp:
+            tree = tree_fp.read()
+            self.assertTrue(tree.endswith("'k__Bacteria':0.0):0.0;\n"))
```

Comment:
The root gets a branch length?

Reply:
I was as surprised as you are but the code above does that ... thoughts?

Reply:
limit the traversal to avoid the root? `tree.traverse(include_self=False)` or `tree.preorder(include_self=False)`, etc

Reply:
changing, thanks!